test_matching_vectors.py

```
@@ -0,0 +1,224 @@
+
+실행 방법:
+python test_matching_vectors.py
+"""
+
+import requests
+import json
+
+BASE_URL = "http://localhost:8000/api"
+
+# 실제 JWT 토큰
+TEST_ACCESS_TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMSIsImVtYWlsIjoic2FuZ3dvb0BuYXZlci5jb20iLCJyb2xlIjoidGFsZW50IiwiZXhwIjoxNzYwMzQ2MDIwfQ.QmUCo9-Wd52egwWjdlHG0Z805XMYw7SUIJDFMvD8oAE"
```

Ruff (0.13.3)

Possible hardcoded password assigned to: "TEST_ACCESS_TOKEN"

(S105)